dashboard/test/navigation_drawer_test.dart

```diff
@@ -57,9 +58,8 @@ void main() {
     });
 
     testWidgets('skia perf links opens skia perf url', (WidgetTester tester) async {
-      const MethodChannel urlLauncherChannel = MethodChannel('plugins.flutter.io/url_launcher');
-      final List<MethodCall> log = <MethodCall>[];
-      urlLauncherChannel.setMockMethodCallHandler((MethodCall methodCall) async => log.add(methodCall));
+      final FakeUrlLauncher urlLauncher = FakeUrlLauncher();
```

Comment:
This should get added to `setUp`

Reply:
umm I thought if we move them to setUp then several tests will share the same fake url launcher instance, and therefore multiple urls will be added to the same list that collects the launches. Therefore I was expecting the urlLauncher.launches.single test to always fail. Much to my surprise, after moving them all to setUp, all individual tests pass smoothly. Maybe my understanding is wrong, but I thought there should be corruption between tests since they are sharing the same fake url launcher instance?

Reply:
setUp is per-test. setUpAll is for all tests in the file, and generally should be avoided

Reply:
Thanks for the explanation! everything makes sense now! as for the other trivial parts, async did not work, to Lists() is needed etc. I will just push another commit that have everything sorted out now